packages/bot-web-ui/src/components/dashboard/dashboard.scss

```diff
@@ -495,9 +510,8 @@
     &__media {
         background: var(--general-section-1);
         text-align: center;
-        padding: 0.8rem;
```

Comment:
@maryia-matskevich-deriv since the padding: 0.8rem is removed, can you change the margin-bottom from 0.8rem to 1.6rem please. Thanks